Improve landing page

juanitorduz opened the pull request:
Make the landing page less dense and add a get-started section

<!-- readthedocs-preview pymc-marketing start -->
----
📚 Documentation preview 📚: https://pymc-marketing--1023.org.readthedocs.build/en/1023/

<!-- readthedocs-preview pymc-marketing end -->

drbenvincent reviewed (changes requested):
Just some quick observations / change requests:
* "Powered by" should be centred along with the logo
* MMM table. Great, but can we make it "Feature" and "Benefit" instead of description.
* I like the illusion of some badges

wd60622 commented on docs/source/getting_started/installation/index.md:
```diff
@@ -0,0 +1,37 @@
+## Installing PyMC-Marketing
```
Can you note installing from GitHub directly via pip for use of latest functionality

juanitorduz replied:
done!

juanitorduz commented:
Thanks @drbenvincent ! I addressed all of your comments 🙏 (we can do something similar for CausalPy if you want)

drbenvincent commented:
That would be great 👍🏻

wd60622 commented on docs/source/contributing/index.md:
````diff
@@ -0,0 +1,153 @@
+
+1. Clone your fork of the `pymc-marketing` repo from your GitHub account to your local disk, and add the base repository as a remote:
+
+    ```bash
+    git clone git@github.com:<your GitHub handle>/pymc-marketing.git
+    cd pymc-marketing
+    git remote add upstream git@github.com:pymc-labs/pymc-marketing.git
````
If you use the `GitHub` CLI, then the required command is just `gh repo fork pymc-labs/pymc-marketing` 😄

juanitorduz replied:
Added in https://github.com/pymc-labs/pymc-marketing/pull/1023/commits/efc8296d5665943e446c0c0d8b3c7cf2f6583760

wd60622 commented:
What's in the left side bar of this page is different than that in the content

![different-bar](https://github.com/user-attachments/assets/52724c64-0764-42bf-9fc7-fdebcfbeaa6d)

juanitorduz commented:
> What's in the left side bar of this page is different than that in the content
> 
> ![different-bar](https://private-user-images.githubusercontent.com/57733339/367330398-52724c64-0764-42bf-9fc7-fdebcfbeaa6d.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMzczNTksIm5iZiI6MTcyNjIzNzA1OSwicGF0aCI6Ii81NzczMzMzOS8zNjczMzAzOTgtNTI3MjRjNjQtMDc2NC00MmJmLTlmYzctZmRlYmNmYmVhYTZkLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTEzVDE0MTczOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU4OGYxYzRlMjQyMDliZDk4YmFkMzBlMjRmMDQ3YmRkYTRlZGIyYTgwZWY0ZDExODA5MTdiYjM0OTg5YjI0ZmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Qx9ss0xSHARy3w1ySvC7Egs28jDG0YQWRdFOWmQ0FhI)

OI would need to create a folder for each section and seems a bit cumbersome . Should I do it 🤔 ?

wd60622 commented:
Your call. I think all the content is good!

OriolAbril commented:
> OI would need to create a folder for each section and seems a bit cumbersome . Should I do it 🤔 ?

I think it has to do with multiple `# title` in the same file, no need for different folders but yes different files if you want different entries in the sidebar. Not sure why, but the sidebar will only show the first entry in each file but the inbody toctree shows all top level titles (independently of them being in the same or different files)

wd60622 commented on docs/source/getting_started/installation/index.md:
````diff
@@ -0,0 +1,51 @@
+pip install git+https://github.com/pymc-labs/pymc-marketing.git
+```
+
+Alternatively, if you use the [GitHub CLI](https://cli.github.com/), then the required command is just
+
+```bash
+gh repo fork pymc-labs/pymc-marketing
+```
````
Wrong place for this. Would be with contribution as alternative to git clone

juanitorduz replied:
Right! Thanks! Fixed inhttps://github.com/pymc-labs/pymc-marketing/pull/1023/commits/e8c6eed87f8310e1aaf470d9b76bba8b8863fcbb

cluhmann commented on docs/source/index.md:
```diff
@@ -106,140 +147,55 @@ Explore our detailed CLV examples using data from the [`lifetimes`](https://gith
-### JupyterLab Notebook
 
-After installing the `pymc-marketing` package (see above), run the following with `marketing_env` activated:
+## Resources
```
Can this be centered? There is quite a bit of content listed here and all the "sections" sort of run together right now.

juanitorduz replied:
Fixed in https://github.com/pymc-labs/pymc-marketing/pull/1023/commits/926f5b545252dc64f0f15a75617390401787d98d

cluhmann commented on docs/source/index.md:
```diff
@@ -106,140 +147,55 @@ Explore our detailed CLV examples using data from the [`lifetimes`](https://gith
-- **Build Within PyMC-Marketing**: Our team are experts leveraging the capabilities of PyMC-Marketing to create robust marketing models for precise insights.
-- **SLA & Coaching**: Get guaranteed support levels and personalized coaching to ensure your team is well-equipped and confident in using our tools and approaches.
-- **SaaS Solutions**: Harness the power of our state-of-the-art software solutions to streamline your data-driven marketing initiatives.
+### More [PyMC-Labs](https://www.pymc-labs.com/) Blogposts and Resources
```
Also, if we can manage to center the "Resources" above, the same treatment here would be useful. It's not obvious that this is a "section" heading.

cluhmann commented on docs/source/getting_started/installation/index.md:
````diff
@@ -0,0 +1,45 @@
+pip install git+https://github.com/pymc-labs/pymc-marketing.git
+```
+
+## Notebooks
````
The heading here is very mysterious. This is not really about "notebooks" but rather about installing jupyterlab/ipykernel so that the notebooks are runnable in the environment.  It's fine as a section heading, but this heading is then used to populate the index page on the "Getting Started" page and the link just says "Notebooks" and it's not clear what you will get when you click on it.

juanitorduz replied:
Simplified it in in https://github.com/pymc-labs/pymc-marketing/pull/1023/commits/926f5b545252dc64f0f15a75617390401787d98d

juanitorduz commented:
> I still think the landing page has a huge amount of information on it (e.g., the page is very long), but at least there is a bit more Labs content and it is nearer to the top of the page now.

@cluhmann Agree. Do you think some of the videos can be removed? Which ones?

cluhmann commented:
> @cluhmann Agree. Do you think some of the videos can be removed? Which ones?

Maybe the "Schedule a Free Consultation for MMM & CLV Strategy" and "Support" sections could be trimmed (I can make concrete suggestions).

The "Examples" under the CLV section (the 2x2 table) doesn't seem to do anything? Is this supposed to include links?

Then, under "Resources" I think we can maybe have 1 video and then a link to a page with the rest of the videos? Or to the playlist on YouTube?

juanitorduz commented on docs/source/index.md:
```diff
@@ -8,12 +8,53 @@
 :::
 
-# PyMC-Marketing - Open Source Marketing Analytics Solution
+# Open Source Marketing Analytics Solution
```
Added in https://github.com/pymc-labs/pymc-marketing/pull/1023/commits/926f5b545252dc64f0f15a75617390401787d98d

juanitorduz commented on docs/source/index.md:
```diff
@@ -8,12 +8,53 @@
-# PyMC-Marketing - Open Source Marketing Analytics Solution
+# Open Source Marketing Analytics Solution
+
+[![Downloads](https://static.pepy.tech/badge/pymc-marketing)](https://pepy.tech/project/pymc-marketing)
+[![Downloads](https://static.pepy.tech/badge/pymc-marketing/month)](https://pepy.tech/project/pymc-marketing)
+[![Downloads](https://static.pepy.tech/badge/pymc-marketing/week)](https://pepy.tech/project/pymc-marketing)
```
I tried but did not find a way (yet)

OriolAbril replied:
to center:

```markdow
:::{div} sd-text-center
[![Downloads](https://static.pepy.tech/badge/pymc-marketing)](https://pepy.tech/project/pymc-marketing)
[![Downloads](https://static.pepy.tech/badge/pymc-marketing/month)](https://pepy.tech/project/pymc-marketing)
[![Downloads](https://static.pepy.tech/badge/pymc-marketing/week)](https://pepy.tech/project/pymc-marketing)
:::
```

juanitorduz replied:
Done! Thanks!

juanitorduz commented:
@cluhmann I actually removed the Support section as it seemed redundant. I left just three videos: a business case, an mmm and a clv one, WDYT?

Besides that, I think I have addressed all your comments.

cluhmann commented:
Let me just do some editing on the "Schedule a Free Consultation" section. Should get to it in a few hours.

cluhmann commented on docs/source/index.md:
```diff
@@ -8,12 +8,51 @@
+
+---
+
+## 📞 Schedule a Free Consultation for MMM & CLV Strategy
```
```suggestion
## 📞 Schedule a Free Strategy Consultation 
```

cluhmann commented on docs/source/index.md:
```diff
@@ -8,12 +8,51 @@
+
+Maximize your marketing ROI with a [free 30-minute strategy session](https://calendly.com/niall-oulton) with our PyMC-Marketing experts. Learn how Bayesian Marketing Mix Modeling and Customer Lifetime Value analytics can boost your organization by making smarter, data-driven decisions.
+
+For businesses looking to integrate PyMC-Marketing into their operational framework, [PyMC Labs](https://www.pymc-labs.com) offers expert consulting and training. Our team is proficient in state-of-the-art Bayesian modeling techniques, with a focus on Marketing Mix Models (MMMs) and Customer Lifetime Value (CLV). Explore these topics further by watching our video on [Bayesian Marketing Mix Models ...
```
```suggestion
For businesses looking to integrate PyMC-Marketing into their operational framework, [PyMC Labs](https://www.pymc-labs.com) offers expert consulting and training. Our team is proficient in state-of-the-art Bayesian modeling techniques, with a focus on Marketing Mix Models (MMMs) and Customer Lifetime Value (CLV).
```

cluhmann commented on docs/source/index.md:
```diff
@@ -8,12 +8,51 @@
+
+We provide the following professional services:
+
+- **Custom Models**: We tailor niche marketing anayltics models to fit your organization's unique needs.
```
```suggestion
- **Custom Models**: We develop models that fit your organization's unique needs.
```

cluhmann commented on docs/source/index.md:
```diff
@@ -8,12 +8,51 @@
+We provide the following professional services:
+
+- **Custom Models**: We tailor niche marketing anayltics models to fit your organization's unique needs.
+- **Build Within PyMC-Marketing**: Our team are experts leveraging the capabilities of PyMC-Marketing to create robust marketing models for precise insights.
```
I would remove this bullet.

juanitorduz replied:
Removed in https://github.com/pymc-labs/pymc-marketing/pull/1023/commits/bdab3a27c03aec92c4a6b090806ed8b607ffcaa8

cluhmann commented on docs/source/index.md:
```diff
@@ -8,12 +8,51 @@
+
+- **Custom Models**: We tailor niche marketing anayltics models to fit your organization's unique needs.
+- **Build Within PyMC-Marketing**: Our team are experts leveraging the capabilities of PyMC-Marketing to create robust marketing models for precise insights.
+- **SLA & Coaching**: Get guaranteed support levels and personalized coaching to ensure your team is well-equipped and confident in using our tools and approaches.
```
```suggestion
- **Coaching**: Regular, personalized coaching to ensure your team is well-equipped to confidently use PyMC-Marketing and related approaches.
```

cluhmann commented on docs/source/index.md:
```diff
@@ -99,147 +138,55 @@ Explore our detailed CLV examples using data from the [`lifetimes`](https://gith
+- [Improving the Speed and Accuracy of Bayesian Media Mix Models](https://www.pymc-labs.com/blog-posts/reducing-customer-acquisition-costs-how-we-helped-optimizing-hellofreshs-marketing-budget/)
+- [Bayesian Media Mix Modeling for Marketing Optimization](https://www.pymc-labs.com/blog-posts/bayesian-media-mix-modeling-for-marketing-optimization/)
+- [Bayesian inference at scale: Running A/B tests with millions of observations](https://www.pymc-labs.com/blog-posts/bayesian-inference-at-scale-running-ab-tests-with-millions-of-observations/)
 
```
```suggestion
For more blogposts and resources, check out the [PyMC Labs Blog](https://www.pymc-labs.com/blog-posts/).

```

cluhmann reviewed (commented):
Let's see how these changes look. Not as compact as I had hoped, but a bit better.

juanitorduz commented:
Thanks @cluhmann ! I think is ready for a last review :)

cluhmann commented on docs/source/index.md:
```diff
@@ -99,147 +137,56 @@ Explore our detailed CLV examples using data from the [`lifetimes`](https://gith
+- [Improving the Speed and Accuracy of Bayesian Media Mix Models](https://www.pymc-labs.com/blog-posts/reducing-customer-acquisition-costs-how-we-helped-optimizing-hellofreshs-marketing-budget/)
+- [Bayesian Media Mix Modeling for Marketing Optimization](https://www.pymc-labs.com/blog-posts/bayesian-media-mix-modeling-for-marketing-optimization/)
+- [Bayesian inference at scale: Running A/B tests with millions of observations](https://www.pymc-labs.com/blog-posts/bayesian-inference-at-scale-running-ab-tests-with-millions-of-observations/)
+For more blogposts and resources, check out the [PyMC Labs Blog](https://www.pymc-labs.com/blog-posts/).
```
Just need an extra blank line so that this last line appears on a new line.

juanitorduz replied:
Good catch! Fixed in https://github.com/pymc-labs/pymc-marketing/pull/1023/commits/f8f1af230f0e22294200bbc32c6fcda5ab440e7e

cluhmann reviewed (approved):
Just the 1 fix I see.
